.storybook/config.js

```diff
@@ -10,7 +10,7 @@ addDecorator(withOptions({
 }));
 
 function loadStories() {
-  const req = require.context('../src/components', true, /\-story\.jsx$/);
+  const req = require.context('../src/components', true, /\.story\.jsx$/);
```

Comment:
So they prefer story as more idiomatic then spec in storybook?

Reply:
We actually have both stories and specs. I changed this to a dot instead of hyphen here, to be consistent between the two. For example:

- input.spec.js
- input.story.js

Instead of one having a dot and one having a hyphen